modules/entity_version_workflows/README.md

```diff
@@ -15,3 +15,7 @@ by setting the custom property "entity_version_no_update" to TRUE.
 ```
 $entity->entity_version_no_update = TRUE;
 ```
+
+## Configurations
+Entity version settings under path "admin/config/entity-version/settings" have to be configured in order to
+have the entity version workflows functionality fully operational.
```

Comment:
Let's add also here that if no field is configured, the workflow functionality is not applied.